aws/modules/iam_role/variables.tf

```diff
@@ -21,3 +21,9 @@ variable "create" {
 	default     = true
 	description = "Set to false to prevent the module from creating any resources"
 }
+
+variable "wait_time" {
+	type = string
+	default = "5s"
```

Comment:
Can we change the default to be 10s? I think we want to make increase the default so that others won't have to customize it that often.

Reply:
Done!

Reply:
Oops, missed the readme bit. Just pushed that as well